Added link further explaining ERB in the 'Getting Started' documentation [ci skip]

mguan2020 opened the pull request:
<!--
Thanks for contributing to Rails!

Please do not make *Draft* pull requests, as they still send
notifications to everyone watching the Rails repo.

Create a pull request when it is ready for review and feedback
from the Rails team :).

If your pull request affects documentation or any non-code
changes, guidelines for those changes are [available
here](https://edgeguides.rubyonrails.org/contributing_to_ruby_on_rails.html#contributing-to-the-rails-documentation)

About this template

The following template aims to help contributors write a good description for their pull requests.
We'd like you to provide a description of the changes in your pull request (i.e. bugs fixed or features added), motivation behind the changes, and complete the checklist below before opening a pull request.

Feel free to discard it if you need to (e.g. when you just fix a typo). -->

### Motivation / Background

<!--
Describe why this Pull Request needs to be merged. What bug have you fixed? What feature have you added? Why is it important?
If you are fixing a specific issue, include "Fixes #ISSUE" (replace with the issue number, remove the quotes) and the issue will be linked to this PR.
-->

This Pull Request has been created because it would be very helpful to provide a link that explains ERB in more depth for people learning the Rails framework. ERB is a very essential part of a Rails application.

### Detail

This Pull Request adds the important fact that ERB stands for "Embedded Ruby." This helps readers better understand the meaning of ERB. This Pull Request also provides a link to a very useful Wikipedia page explaining ERB in more depth for readers who want to explore the topic in more depth.

### Additional information

<!-- Provide additional information such as benchmarks, reference to other repositories or alternative solutions. -->

### Checklist

Before submitting the PR make sure the following are checked:

* [X] This Pull Request is related to one change. Changes that are unrelated should be opened in separate PRs.
* [X] Commit message has a detailed description of what changed and why. If this PR fixes a related issue include it in the commit message. Ex: `[Fix #issue-number]`
* [X] Tests are added or updated if you fix a bug or add a feature.
* [X] CHANGELOG files are updated for the changed libraries if there is a behavior change or additional feature. Minor bug fixes and documentation changes should not be included.

mguan2020 committed:
Add link providing more details about ERB in the 'Getting Started' documentation [ci skip]

Co-Authored-By: Akhil G Krishnan <akhilgkrishnan4u@gmail.com>

Add link providing more details about ERB in the 'Getting Started' documentation [ci skip]

Co-Authored-By: Akhil G Krishnan <akhilgkrishnan4u@gmail.com>

Fix link to point to html [ci skip]

Co-authored-by: Rafael Mendonça França <rafael@franca.dev>

Revert "Merge branch 'branch_new' of https://github.com/mguan2020/rails into pr/49640"

This reverts commit 94b8c7d579a74c528c4a85f3ce0f0853c4883fd1, reversing
changes made to 1353ca902f260dfb9bbe7680ede2650a670f659f.

Revert "Revert "Merge branch 'branch_new' of https://github.com/mguan2020/rails into pr/49640""

This reverts commit e9ced85c8f6afe0bc057bc6d8d4387718a7c3916.

akhilgkrishnan commented on guides/source/getting_started.md:
````diff
@@ -558,7 +558,7 @@ file, and replace its contents with:
 ```
 
-The above code is a mixture of HTML and *ERB*. ERB is a templating system that
+The above code is a mixture of HTML and *ERB*. ERB, short for [Embedded Ruby](https://en.wikipedia.org/wiki/ERuby#:~:text=Embedded%20Ruby%20), is a templating system that
````
```suggestion
The above code is a mixture of HTML and *ERB*. ERB, short for [Embedded Ruby](https://en.wikipedia.org/wiki/ERuby), is a templating system that
```

mguan2020 replied:
Great suggestion! I'll go ahead and make the change.

akhilgkrishnan reviewed (commented):
Could you please do a [squash commit](https://guides.rubyonrails.org/contributing_to_ruby_on_rails.html#squashing-commits) and amend the commit message with `[skip ci]`

mguan2020 commented:
Hi @akhilgkrishnan,

I am having some issues with squashing my commits. I am getting the error "error: failed to push some refs to 'https://github.com/mguan2020/rails.git" after I run the command 'git push fork branch_new --force-with-lease'.
I have carefully read the guide on Contributing to Ruby on Rails.

Would it be possible for you to help? Thank you very much for your time.

akhilgkrishnan commented:
> Hi @akhilgkrishnan,
> 
> 
> 
> I am having some issues with squashing my commits. I am getting the error "error: failed to push some refs to 'https://github.com/mguan2020/rails.git" after I run the command 'git push fork branch_new --force-with-lease'.
> 
> I have carefully read the guide on Contributing to Ruby on Rails.
> 
> 
> 
> Would it be possible for you to help? Thank you very much for your time.

Seems like you haven't taken the latest pull. Take the pull first and then do the squashing.

mguan2020 commented:
Hi @akhilgkrishnan,

Thank you for your advice. I have squashed the commits into one single commit.

Please let me know if there is anything else I need to do.

akhilgkrishnan commented:
> Hi @akhilgkrishnan,
> 
> 
> 
> Thank you for your advice. I have squashed the commits into one single commit.
> 
> 
> 
> Please let me know if there is anything else I need to do.


I think the [skip ci] still missing on your commit message. So could you please amend your commit.

mguan2020 commented:
> > Hi @akhilgkrishnan,
> > Thank you for your advice. I have squashed the commits into one single commit.
> > Please let me know if there is anything else I need to do.
> 
> I think the [skip ci] still missing on your commit message. So could you please amend your commit.

Hi @akhilgkrishnan,

Thank you for the reminder. I have appended [skip ci] to my commit message.

akhilgkrishnan commented:
> > > Hi @akhilgkrishnan,
> > > Thank you for your advice. I have squashed the commits into one single commit.
> > > Please let me know if there is anything else I need to do.
> > 
> > 
> > I think the [skip ci] still missing on your commit message. So could you please amend your commit.
> 
> Hi @akhilgkrishnan,
> 
> Thank you for the reminder. I have appended [skip ci] to my commit message.

I don't see it 🤔

mguan2020 commented:
> > Hi @akhilgkrishnan,
> > Thank you for the reminder. I have appended [skip ci] to my commit message.
> 
> I don't see it 🤔

Apologies for the confusion. It should be there now.

akhilgkrishnan commented:
> > > Hi @akhilgkrishnan,
> > > Thank you for the reminder. I have appended [skip ci] to my commit message.
> > 
> > 
> > I don't see it 🤔
> 
> My apologies for the confusion. It should be there now.

No apologies please.

akhilgkrishnan reviewed (approved):
Thanks @mguan2020 👍

zzak commented:
@akhilgkrishnan We are not particularly strict about adding `ci skip` to doc commits, fyi ❤️

zzak reviewed (approved):
The line wrapping is throwing me off but I guess it's fine :joy:

rafaelfranca commented on guides/source/getting_started.md:
````diff
@@ -558,7 +558,7 @@ file, and replace its contents with:
 ```
 
-The above code is a mixture of HTML and *ERB*. ERB is a templating system that
+The above code is a mixture of HTML and *ERB*. ERB, short for [Embedded Ruby](https://en.wikipedia.org/wiki/ERuby), is a templating system that
````
```suggestion
The above code is a mixture of HTML and *ERB*. ERB, short for [Embedded Ruby](https://docs.ruby-lang.org/en/3.2/ERB.html), is a templating system that
```

akhilgkrishnan commented:
@mguan2020 Could you please update the suggested changes?

mguan2020 commented:
> @mguan2020 Could you please update the suggested changes?

Thank you very much for the reminder. I have incorporated @rafaelfranca suggested changes. All of the commits are now squashed into one.

Could you please review the PR again? Thank you for your time.

akhilgkrishnan commented:
> > @mguan2020 Could you please update the suggested changes?
> 
> Thank you very much for the reminder. I have incorporated @rafaelfranca suggested changes. All of the commits are now squashed into one.
> 
> Could you please review the PR again? Thank you for your time.

Looks good to me 👍
